### PR TITLE
Fixed some tests from python-505 branch

### PR DESCRIPTION
**Pull request by bjmb**

### CODE REVIEW

**Comment by mambocab** [comment]
LGTM if the tests pass, but I'd like a review from Alan as well -- @aboudreault, could you have a look please?

File: cassandra/cqlengine/columns.py [hunk 1]
```diff
@@ -579,6 +579,15 @@ def to_database(self, value):
         d = value if isinstance(value, util.Date) else util.Date(value)
         return d.days_from_epoch + SimpleDateType.EPOCH_OFFSET_DAYS
 
+    def to_python(self, value):
+        value = super(Date, self).to_database(value)
```
Comment (aboudreault) [inline cassandra/cqlengine/columns.py]: I think we can remove this

File: cassandra/cqlengine/columns.py [hunk 2]
```diff
@@ -597,6 +606,13 @@ def to_database(self, value):
         # str(util.Time) yields desired CQL encoding
         return value if isinstance(value, util.Time) else util.Time(value)
 
+    def to_python(self, value):
+        value = super(Time, self).to_database(value)
```
Comment (aboudreault) [inline cassandra/cqlengine/columns.py]: I think we can remove this

**Comment by bjmb** [comment]
I'll merge this and keep the python-505 open because some tests (the on related with UDTs) are still failing

**Pull request merged**